modules/mesh-task/main.tf

```diff
@@ -46,10 +46,6 @@ locals {
           concat(
             lookup(def, "dependsOn", []),
             [
-              {
-                containerName = "consul-ecs-control-plane"
-                condition     = "HEALTHY"
-              },
               {
```

Comment:
Should mesh-init be added here?

Reply:
It's transitive. `dataplane` can only be healthy if `mesh-init` is successful and so this dependsOn block implicitly depends on `mesh-init`